### PR TITLE
K8SPG-196: Allow exposing superusers through pgBouncer

### PR DESCRIPTION
**Pull request by egegunes**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/6ab35722/init-deploy.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/6ab35722/demand-backup.log) |
| upgrade | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/6ab35722/upgrade.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/6ab35722/scaling.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/6ab35722/smart-update.log) |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/6ab35722/recreate.log) |
| version-service | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/6ab35722/version-service.log) |
| affinity | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/6ab35722/affinity.log) |
| scheduled-backup | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/6ab35722/scheduled-backup.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/6ab35722/monitoring.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/6ab35722/self-healing.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/6ab35722/operator-self-healing.log) |
| clone-cluster | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/6ab35722/clone-cluster.log) |
| tls-check | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/6ab35722/tls-check.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/6ab35722/users.log) |
| ns-mode | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/6ab35722/ns-mode.log) |
| We run 16 out of 16|

commit: https://github.com/percona/percona-postgresql-operator/pull/270/commits/6ab357228b3a00e87a06262fc0e10271c6dabfdf
image: `perconalab/percona-postgresql-operator:PR-270-6ab35722-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-270-6ab35722-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-270-6ab35722-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-270-6ab35722-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-270-6ab35722-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-270-6ab35722-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/08b87edd/demand-backup.log) |
| upgrade | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/08b87edd/upgrade.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/08b87edd/init-deploy.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/08b87edd/smart-update.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/08b87edd/scaling.log) |
| version-service | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/08b87edd/version-service.log) |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/08b87edd/recreate.log) |
| scheduled-backup | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/08b87edd/scheduled-backup.log) |
| affinity | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/08b87edd/affinity.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/08b87edd/monitoring.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/08b87edd/self-healing.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/08b87edd/operator-self-healing.log) |
| clone-cluster | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/08b87edd/clone-cluster.log) |
| tls-check | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/08b87edd/tls-check.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/08b87edd/users.log) |
| ns-mode | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/08b87edd/ns-mode.log) |
| We run 16 out of 16|

commit: https://github.com/percona/percona-postgresql-operator/pull/270/commits/08b87edd0acf91eaf9f9b7e6c7f901a46b1dfc1e
image: `perconalab/percona-postgresql-operator:PR-270-08b87edd-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-270-08b87edd-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-270-08b87edd-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-270-08b87edd-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-270-08b87edd-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-270-08b87edd-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/08b87edd/init-deploy.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/08b87edd/scaling.log) |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/08b87edd/recreate.log) |
| affinity | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/08b87edd/affinity.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/08b87edd/demand-backup.log) |
| upgrade | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/08b87edd/upgrade.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/08b87edd/monitoring.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/08b87edd/smart-update.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/08b87edd/scheduled-backup.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/08b87edd/self-healing.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/08b87edd/version-service.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/08b87edd/operator-self-healing.log) |
| clone-cluster | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/08b87edd/clone-cluster.log) |
| tls-check | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/08b87edd/tls-check.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/08b87edd/users.log) |
| ns-mode | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/08b87edd/ns-mode.log) |
| We run 16 out of 16|

commit: https://github.com/percona/percona-postgresql-operator/pull/270/commits/08b87edd0acf91eaf9f9b7e6c7f901a46b1dfc1e
image: `perconalab/percona-postgresql-operator:PR-270-08b87edd-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-270-08b87edd-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-270-08b87edd-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-270-08b87edd-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-270-08b87edd-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-270-08b87edd-pgo-deployer`

**Review by hors — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/eec7e877/init-deploy.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/eec7e877/demand-backup.log) |
| upgrade | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/eec7e877/upgrade.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/eec7e877/smart-update.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/eec7e877/scaling.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/eec7e877/version-service.log) |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/eec7e877/recreate.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/eec7e877/scheduled-backup.log) |
| affinity | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/eec7e877/affinity.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/eec7e877/monitoring.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/eec7e877/self-healing.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/eec7e877/operator-self-healing.log) |
| clone-cluster | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/eec7e877/clone-cluster.log) |
| tls-check | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/eec7e877/tls-check.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/eec7e877/users.log) |
| ns-mode | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/eec7e877/ns-mode.log) |
| We run 16 out of 16|

commit: https://github.com/percona/percona-postgresql-operator/pull/270/commits/eec7e877dc458f92fdc52ec598ccf5f391bdf69e
image: `perconalab/percona-postgresql-operator:PR-270-eec7e877-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-270-eec7e877-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-270-eec7e877-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-270-eec7e877-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-270-eec7e877-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-270-eec7e877-pgo-deployer`

**Review by hors — APPROVED** [review]

File: deploy/cr.yaml [hunk 1]
```diff
@@ -116,6 +116,7 @@ spec:
   pgBouncer:
     image: perconalab/percona-postgresql-operator:main-ppg14-pgbouncer
 #    imagePullPolicy: Always
+    exposeSuperusers: false
```
[the remark this answers is not in the document]
Comment (hors) [reply deploy/cr.yaml]: @cap1984 please add test case

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/eec7e877/demand-backup.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/eec7e877/scheduled-backup.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/eec7e877/init-deploy.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/eec7e877/scaling.log) |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/eec7e877/recreate.log) |
| affinity | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/eec7e877/affinity.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/eec7e877/monitoring.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/eec7e877/self-healing.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/eec7e877/operator-self-healing.log) |
| clone-cluster | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/eec7e877/clone-cluster.log) |
| upgrade | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/eec7e877/upgrade.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/eec7e877/smart-update.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/eec7e877/version-service.log) |
| tls-check | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/eec7e877/tls-check.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/eec7e877/users.log) |
| ns-mode | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/eec7e877/ns-mode.log) |
| We run 16 out of 16|

commit: https://github.com/percona/percona-postgresql-operator/pull/270/commits/eec7e877dc458f92fdc52ec598ccf5f391bdf69e
image: `perconalab/percona-postgresql-operator:PR-270-eec7e877-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-270-eec7e877-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-270-eec7e877-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-270-eec7e877-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-270-eec7e877-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-270-eec7e877-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| upgrade | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/33d72c00/upgrade.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/33d72c00/demand-backup.log) |
| init-deploy | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/33d72c00/init-deploy.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/33d72c00/smart-update.log) |
| version-service | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/33d72c00/version-service.log) |
| scheduled-backup | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/33d72c00/scheduled-backup.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/33d72c00/scaling.log) |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/33d72c00/recreate.log) |
| affinity | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/33d72c00/affinity.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/33d72c00/monitoring.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/33d72c00/self-healing.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/33d72c00/operator-self-healing.log) |
| clone-cluster | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/33d72c00/clone-cluster.log) |
| tls-check | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/33d72c00/tls-check.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/33d72c00/users.log) |
| ns-mode | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/33d72c00/ns-mode.log) |
| We run 16 out of 16|

commit: https://github.com/percona/percona-postgresql-operator/pull/270/commits/33d72c00b400510eb00385e13cbb805d28f72467
image: `perconalab/percona-postgresql-operator:PR-270-33d72c00-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-270-33d72c00-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-270-33d72c00-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-270-33d72c00-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-270-33d72c00-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-270-33d72c00-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/45f746bf/init-deploy.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/45f746bf/demand-backup.log) |
| upgrade | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/45f746bf/upgrade.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/45f746bf/smart-update.log) |
| version-service | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/45f746bf/version-service.log) |
| scheduled-backup | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/45f746bf/scheduled-backup.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/45f746bf/scaling.log) |
| recreate | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/45f746bf/recreate.log) |
| affinity | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/45f746bf/affinity.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/45f746bf/monitoring.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/45f746bf/self-healing.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/45f746bf/operator-self-healing.log) |
| clone-cluster | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/45f746bf/clone-cluster.log) |
| tls-check | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/45f746bf/tls-check.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/45f746bf/users.log) |
| ns-mode | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/45f746bf/ns-mode.log) |
| We run 16 out of 16|

commit: https://github.com/percona/percona-postgresql-operator/pull/270/commits/45f746bfce7c6a5fa8fa52c57a3c00818065fc0a
image: `perconalab/percona-postgresql-operator:PR-270-45f746bf-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-270-45f746bf-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-270-45f746bf-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-270-45f746bf-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-270-45f746bf-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-270-45f746bf-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/4b5df4fd/init-deploy.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/4b5df4fd/demand-backup.log) |
| upgrade | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/4b5df4fd/upgrade.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/4b5df4fd/smart-update.log) |
| version-service | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/4b5df4fd/version-service.log) |
| scheduled-backup | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/4b5df4fd/scheduled-backup.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/4b5df4fd/scaling.log) |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/4b5df4fd/recreate.log) |
| affinity | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/4b5df4fd/affinity.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/4b5df4fd/monitoring.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/4b5df4fd/self-healing.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/4b5df4fd/operator-self-healing.log) |
| clone-cluster | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/4b5df4fd/clone-cluster.log) |
| tls-check | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/4b5df4fd/tls-check.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/4b5df4fd/users.log) |
| ns-mode | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/4b5df4fd/ns-mode.log) |
| We run 16 out of 16|

commit: https://github.com/percona/percona-postgresql-operator/pull/270/commits/4b5df4fda1ed6b0272df469a26772bd3c517f39f
image: `perconalab/percona-postgresql-operator:PR-270-4b5df4fd-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-270-4b5df4fd-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-270-4b5df4fd-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-270-4b5df4fd-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-270-4b5df4fd-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-270-4b5df4fd-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/62cbf394/demand-backup.log) |
| upgrade | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/62cbf394/upgrade.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/62cbf394/init-deploy.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/62cbf394/smart-update.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/62cbf394/scaling.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/62cbf394/version-service.log) |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/62cbf394/recreate.log) |
| scheduled-backup | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/62cbf394/scheduled-backup.log) |
| affinity | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/62cbf394/affinity.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/62cbf394/monitoring.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/62cbf394/self-healing.log) |
| operator-self-healing | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/62cbf394/operator-self-healing.log) |
| clone-cluster | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/62cbf394/clone-cluster.log) |
| tls-check | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/62cbf394/tls-check.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/62cbf394/users.log) |
| ns-mode | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-270/62cbf394/ns-mode.log) |
| We run 16 out of 16|

commit: https://github.com/percona/percona-postgresql-operator/pull/270/commits/62cbf394a09a9c65b34b4fb9e84e91d1ecbf6f96
image: `perconalab/percona-postgresql-operator:PR-270-62cbf394-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-270-62cbf394-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-270-62cbf394-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-270-62cbf394-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-270-62cbf394-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-270-62cbf394-pgo-deployer`

**Review by cap1984 — APPROVED** [review]

**Pull request merged**